lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1794,6 +1796,64 @@ def configure_disks(self, vm_obj):
             if disk_index == 7:
                 disk_index += 1
 
```

Comment:
This new block (and a bit of the old code after) should be in a new `attach_disk()` function.